### ISSUE TITLE
Music triggers are not working properly

### ISSUE THREAD

**Issue by ArchDuque-Pancake** [yairm210/Unciv#7392]
In my mod of Latin American Civs I had recently added some main themes to each civ, and they play correctly when it should with the "- Theme" suffix.

So it is working as intended here.
But, when playing normaly, other nations themes start playing without any trigger, just randomly, even if say Brazil isn't in the game, the Theme can still play as it were and ambient song.

And this is not a problem that 'there is no other music, so it takes whatever song there is to not repeat the same slng', no, I have the tatched villagers, on my phone another ambient music pack, and on PC the Victoria music and it is still hapening.

Tldr: Themes from other nations play randomly without actual trigger.

**Comment by AdityaMH** [comment]
To fix it: Just add additional music with "ambient" suffix

**Comment by ArchDuque-Pancake** [comment]
As I've said, I have more music apart from the themes, plenty which "Ambient" Suffix in it

**Comment by ArchDuque-Pancake** [comment]
And its not just with my mod, it also happens with "Unciv Vainilla music pack"
